site/_scss/globals/_colors.scss

```diff
@@ -91,4 +91,20 @@ $google-colors: (
   'cyan-200': #a1e4f2,
   'cyan-100': #cbf0f8,
   'cyan-50': #e4f7fb,
+  'heart-red': #f44336,
```

Comment:
If these styles are specific to Chrome 100, they should not be a in global style definition.